R/check_forecasts.R

```diff
@@ -293,7 +289,7 @@ print.scoringutils_check <- function(x, ...) {
 find_duplicates <- function(data, forecast_unit, ...) {
   type <- c("sample", "quantile")[c("sample", "quantile") %in% colnames(data)]
   if (missing(forecast_unit)) {
-     forecast_unit <- get_forecast_unit(data, ...)
+    forecast_unit <- get_forecast_unit(data, ...)
```

Comment:
I think we need to remove `...` here
```suggestion
    forecast_unit <- get_forecast_unit(data)
```

Reply:
Done in https://github.com/epiforecasts/scoringutils/pull/305/commits/354c42e1664759013cdc46f2c72785e96d93e282